test/e2e/conformance/cross_logical_cluster_list_test.go

```diff
@@ -186,7 +188,7 @@ func TestCRDCrossLogicalClusterListPartialObjectMetadata(t *testing.T) {
 	bootstrapCRD(t, wsNormalCRD1b, crdClusterClient.Cluster(wsNormalCRD1b).ApiextensionsV1().CustomResourceDefinitions(), sheriffCRD1)
 
 	t.Logf("Create a root shard client that is able to do wildcard requests")
-	rootCfg := framework.ShardConfig(t, kcpClusterClient, "root", cfg)
+	rootCfg := framework.ShardConfig(t, kcpClusterClient, "root", rootShardConfig)
```

Comment:
can we drop this line?

Reply:
same as the others, used once in a wildcard call: https://github.com/kcp-dev/kcp/blob/8c833d4f55313383b8d8bc3d4069f06822df893d/test/e2e/conformance/cross_logical_cluster_list_test.go#L196

Reply:
rootCfg will be the same as rootShardCfg. ShardConfig on rootShardCfg is a noop.

Reply:
removed